### DISCUSSION TITLE
Multivariate vs. multivariable clarification

### DISCUSSION THREAD

**Discussion by giemmecci** [sebp/scikit-survival#252] [Q&A] [answered]
Hi, thanks for developing this module.

Taking into account the publication ["Multivariate or Multivariable Regression?"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3518362/), could you clarify the term multivariate used when [describing the survival model](https://scikit-survival.readthedocs.io/en/stable/user_guide/00-introduction.html#Multivariate-Survival-Models)?

Thanks

**Comment by sebp** [comment, marked as answer]
According to the definition in the paper it would be multivariable.

**Reply by giemmecci** [reply]
Thanks!
